### ISSUE TITLE
GOG Release

### ISSUE THREAD

**Issue by Mobeeuz** [sucklead/DCoTEPatch#3]
GOG just did a release with some important fixes, such as the invisible sorcerers. However, the walk FOV no longer does anything in this version. Attached is the new EXE...

[CoCMainWin32.zip](https://github.com/sucklead/DCoTEPatch/files/1432824/CoCMainWin32.zip)

**Comment by sucklead** [comment]
Verified that the new shaders that fix the sorcerer issue are from here: https://forums.geforce.com/default/topic/884419/geforce-drivers/call-of-cthulhu-corrupted-graphics-windows-7-fix-/post/4772949/#4772949
I'll add in an option to patch the shaders when I get a chance.
exe is the same size so the xml is in the same place.
There are changes to the exe and launcher exe but not in the xml area so no idea what they change.

**Comment by 07416** [comment]
The game has been removed from GOG: https://www.gog.com/forum/general/release_call_of_cthulhu_dark_corners_of_the_earth_b7169/post203

**Comment by karatekommando** [comment]
Im using the gog version and i get a crash with an error when i apply the patch. 
Application error 0xc0000005.

**Comment by ghost** [comment]
> 
> 
> Im using the gog version and i get a crash with an error when i apply the patch.
> Application error 0xc0000005.

Update the comment with your installer version please.

**Comment by karatekommando** [comment]
> > Im using the gog version and i get a crash with an error when i apply the patch.
> > Application error 0xc0000005.
> 
> Update the comment with your installer version please.
the cocdcotelauncher.exe says its 1.0.0.1.

the readme also states its retail version 1.0.

**Comment by karatekommando** [comment]
Just to add a note, I ended up downloading the .exe above and it now works thanks.

**Comment by ghost** [comment]
@karatekommando Yes, the core game itself necessarily hasn't received any updates, but GOG may have done their own modifications.

`v1.0 (14415)` seems to be the latest version.
